frontend/src/locales/zh-CN.json

```
@@ -1366,6 +1374,10 @@
         "title": "环境级别",
         "desc": "「环境级别」可以将环境标记为受保护的"
       },
+      "bb-feature-lgtm": {
+        "title": "LGTM 检查",
+        "desc": "检查工单是否有「LGTM」评论。"
```

```suggestion
        "desc": "检查工单是否有 \"LGTM\" 评论。"
```
I think「」is for Chinese characters.